docs/aurora/data-science/frameworks/tensorflow.md

```diff
@@ -213,9 +215,66 @@ export MPIR_CVAR_ENABLE_GPU=0
 # This is to disable certain GPU optimizations like the use of XeLinks between
 # GPUs, collectives with GPU-placed data etc., in order to reduce `MPI_Init`
 # overheads. Benefits are application dependent.
+```
```

Comment:
Assuming the next ~100 lines are identical to those in `docs/aurora/data-science/frameworks/pytorch.md`, can you remove these and replace it with `pymdownx.snippets` section reference, as demonstrated in https://github.com/argonne-lcf/user-guides/pull/492#pullrequestreview-2406026019